metadata/images.go

```diff
@@ -55,6 +55,28 @@ func (s *imageStore) Get(ctx context.Context, name string) (images.Image, error)
 
 	if err := view(ctx, s.db, func(tx *bolt.Tx) error {
 		bkt := getImagesBucket(tx, namespace)
+		if bkt == nil {
```

Comment:
Why is this only checking on existence of the images bucket?

Reply:
A mistake ;-(. Thanks for catching this. I updated it to:
```go
if bkt == nil || bkt.Bucket([]byte(name)) == nil {
....
}
```